### DISCUSSION TITLE
paru search

### DISCUSSION THREAD

**Discussion by khagaroth** [Morganamilo/paru#937] [Q&A]
Is it expected that entering several EXACT package names at once (ie paru package1 package2 packageX...) will return an incomplete list (for example completely missing package3) of packages to install?
Note that doing paru -Syu package1 package2 packageX... works correctly.

**Comment by eclairevoyant** [comment]
Yes, see `man paru`:

```
NO OPERATION
       <NO OPTION>
              Interactive search and install. Displays a list of packages matching the search terms and prompts the user on which packages to install.
```

If you want to install specific packages, use `-S` as the operation.
